content/webapp/components/MediaObjectBase/MediaObjectBase.tsx

```diff
@@ -143,17 +141,8 @@ const MediaObjectBase: FunctionComponent<Props> = ({
         {ExtraInfo}
 
         {description && (
-          <div
-            className={classNames({
-              'spaced-text': true,
-              [font('intr', 5)]: !descriptionIsString,
-            })}
-          >
-            {descriptionIsString ? (
-              <p className={font('intr', 5)}>{description}</p>
-            ) : (
-              description
-            )}
```

Comment:
`descriptionIsString` is a boolean:

* if it's true, we apply `font('intr', 5)` on the `<p>` tag in the first branch of the ternary
* if it's false, we apply `font('intr', 5)` on the `<div>` tag using `classNames`

We always apply the `font('intr', 5)` class; doing it in two places feels like a bit of a code smell – we should be able to apply it once on the `<div>` and be done.